src/shaders/fill_extrusion.vertex.glsl

```diff
@@ -158,6 +163,13 @@ void main() {
     v_color.rgb += clamp(color.rgb * directional * u_lightcolor, mix(vec3(0.0), vec3(0.3), 1.0 - u_lightcolor), vec3(1.0));
     v_color *= u_opacity;
 
+    if (!u_rounded_roof) {
```

Comment:
@akoylasar Should we move all of the shader changes under e.g. `#ifdef VERTICAL_EDGE_RADIUS`? The feature is not available on gl-js and it is good not to tax (even though the benchmarks might show no issues) existing code paths that don't use edge radius.

Reply:
I did consider this of course but wanted to avoid yet another ifdef in the shader. I can add this, just to be clear though I will leave the uniform in place. Because we could also use this ifdef to actually generate two flavours of the shader (w/wo support for flat roof edge) and this isn't what you meant here.

Reply:
I decided to introduce an ifdef and remove the uniform. Basically we'll end up with yet another flavour for the shader, the reason is that this way 1. gl-js remains identical to how it was before and 2. the code is just simpler and more performant.